include/zephyr/drivers/bluetooth.h

```diff
@@ -69,7 +69,9 @@ enum bt_hci_bus {
 	BT_HCI_BUS_SDIO          = 6,
 	BT_HCI_BUS_SPI           = 7,
 	BT_HCI_BUS_I2C           = 8,
-	BT_HCI_BUS_IPM           = 9,
+	BT_HCI_BUS_SMD           = 9,
+	BT_HCI_BUS_VIRTIO        = 10,
+	BT_HCI_BUS_IPM           = 11,
```

Comment:
I would recommend instead using `IPC` instead of `IPM`, since `IPM` is a driver class in zephyr, but the subsystem used is the `IPC` one.

Reply:
I change it to IPC, but I also left IPM as an alias, since there are several DT bindings referring to it which I don't feel comfortable touching this close to the release.